### ISSUE TITLE
this plugin requires Mattermost v5.30.0 or later?

### ISSUE THREAD

**Issue by gleke** [mattermost/mattermost-plugin-demo#134]
{"level":"error","ts":1607756624.6742744,"caller":"mlog/log.go:229","msg":"Unable to activate plugin","plugin_id":"com.mattermost.demo-plugin","error":"this plugin requires Mattermost v5.30.0 or later"}

**Comment by hanzei** [comment]
That is correct. The latest version of this plugin requires changes that are only available in the current developers version of Mattermost.
